Assets/Scripts/System/ContextMenuSystem.cs

```diff
@@ -55,6 +73,44 @@ public void OpenSubmenu(Guid menuId, List<ContextMenuState.Placement> possiblePl
             _editorEvents.InvokeUpdateContextMenuEvent();
         }
 
+        /// <summary>
+        /// Open the given context menu and sort all its items by their sorting priotiry.
+        /// </summary>
+        /// <param name="position">The position used as origin for the context menu.</param>
+        /// <param name="items">All the menu items for the context menu.</param>
+        public void OpenMenuSorted(Vector3 position, List<ContextMenuItem> items)
```

Comment:
This is never used. And it calles it self on every path, resulting in an infinite loop when called

Reply:
Infinite recursion is removed.